### ISSUE TITLE
(Minor) "X is Typing..." showing even though nobody's not typing

### ISSUE THREAD

**Issue by Penaz91** [dequis/purple-facebook#297]
As from title, since February 06th, 2017; Pidgin started showing one of my buddies randomly typing to me, even though they're not actually typing anything, I couldn't find anything suggesting anything too look at to help fixing this issue.

Thank you for your time and this great plugin.

**Comment by Malvineous** [comment]
Have you tried using the Facebook app or website?  I have experienced this issue also but both the Android app and the official web client also say the contact is typing when they are not.  It looks like a server-side bug to me.

**Comment by Penaz91** [comment]
Yeah, I started noticing this issue on the Android client too, recently.

Il 08/mar/2017 01:01, "Adam Nielsen" <notifications@github.com> ha scritto:

Have you tried using the Facebook app or website? I have experienced this
issue also but both the Android app and the official web client also say
the contact is typing when they are not. It looks like a server-side bug to
me.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/dequis/purple-facebook/issues/297#issuecomment-284901267>,
or mute the thread
